### PR TITLE
feat(web): add finalized status field to block

### PR DESCRIPTION
**Pull request by luis-herasme**
### Description
Add finalized status field to block

### Related Issue
(https://github.com/Blobscan/blobscan/issues/364)

### Screenshots
image
image
image
image

### CODE REVIEW

**Review by PabloCastellano — COMMENTED** [review]
Can you add some screenshots?

**Review by PabloCastellano — COMMENTED** [review]
Is the button maybe too big?

**Review by xFJA — COMMENTED** [review]
Good job! Just a picky comment regarding the badge: the icon is not fully vertical aligned with the text because they have different heights. The difference is tiny but we could adjust the heights to be totally vertically aligned!

**Comment by xFJA** [comment]
> Is the button maybe too big?

We could reduce some `padding` to adjust the row height and change the `line-height`. This way we can reduce to `24px`the total `height`of the row and we avoid a jump in the list (you can check the screenshots)

I've made a quick draft and looks ok:

image

**Review by xFJA — APPROVED** [review]

File: apps/web/src/components/Badges/StatusBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import { Badge } from "./Badge";
+import type { BadgeProps } from "./Badge";
+
+const statusBadgeVariants = cva(
+  `
+    rounded-lg
+    font-medium
+    text-xs
+    px-[6px]
+    py-[3px]
```
[the remark this answers is not in the document]
Comment (luis-herasme) [reply apps/web/src/components/Badges/StatusBadge.tsx]: Done [50a89cb](https://github.com/Blobscan/blobscan/pull/531/commits/50a89cb8b8a9379fe2b49e142271eb6b26796c8b)

**Review by PabloCastellano — APPROVED** [review]
lgtm!

**Pull request merged**